apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-7_1/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>apm-es-restclient-plugin</artifactId>
+        <groupId>co.elastic.apm</groupId>
+        <version>1.6.2-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>apm-es-restclient-plugin-7_1</artifactId>
+    <name>${project.groupId}:${project.artifactId}</name>
+
+    <properties>
+        <version.elasticsearch>7.1.0</version.elasticsearch>
+        <apm-agent-parent.base.dir>${project.basedir}/../../..</apm-agent-parent.base.dir>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.elasticsearch.client</groupId>
+            <artifactId>elasticsearch-rest-client</artifactId>
+            <version>${version.elasticsearch}</version>
+            <scope>provided</scope>
```

Comment:
Not super important, but everything in this pom can be scoped for `test`

Reply:
changed to test